app/docs/enterprise/0.31-x/kong-architecture-patterns.md

```diff
@@ -0,0 +1,60 @@
+
+## Multi-datacenter with Cassandra
+
+Application availability is important, and high availability can be achieved by configuring Kong in a multi-datacenter setup using [Apache Cassandra](https://getkong.org/about/faq/#apache-cassandra) as the backing datastore. In this configuration each geographically disparate datacenter is host to at least three Kong nodes, and the appropriate number of Cassandra nodes.
+
+The minimum number of Kong nodes per data center needed for high availability is three. This will ensure that in the unlikely event of a node failure there will still be a node and a backup. Cassandra would be configured in an optimal way; you can use the [Cassandra Calculator](https://www.ecyrd.com/cassandracalculator/) to determine how many Cassandra nodes are needed per data center to ensure high availability, and  performance for your specific needs.
+
+Logically Kong will act as a single cluster, and in a multi-datacenter strategy, can be configured similar to the previous architecture patterns. We recommending having the upstream services behind a firewall (or in a VPC in AWS) and only allow Kong nodes direct access to the APIs. 
+
+## Separate Kong Admin node
+
+For security conscious customers we recommend a pattern of having a single node in a Kong cluster allow access to the Admin API. 
+
+Each node in Kong cluster can be configured using its kong.conf file. Two separate configurations can be created, one for a proxy only node, and the other for an admin only node. The proxy only nodes would be accessible from the outside world. They would connect to the database and the upstream APIs that are behind the firewall. The admin only node would be inside the firewall, or a separate firewall (or VPC on AWS), and it would connect to the database that the proxy nodes connects to.
+
+So long as the proxy nodes and admin nodes use the same database they will be part of the same cluster. The admin node itself can be located anywhere. There are a few different ways to achieve this. read [securing the Kong Admin API](https://getkong.org/docs/latest/secure-admin-api/) for details.
+
+### Unlimited options
+
+Kong as a tool is flexible, and offers many possibilities for incorporating it into your existing infrastructure. These patterns we touch upon are the ones we commonly see and recommend. Every situation is unique, and we are happy to assist you with your specific needs.
+
+
+External Links
```

Comment:
seems strange to call this section External links since all but one is on the same domain as the document the reader is viewing. Maybe "Reference Links"? Github link should point to https://github.com/kong/kong, and would change "Kong Documentation" to "Kong CE Documentation" to make it obvious they're ejecting from the enterprise documentation. Also `/install` is actually just a redirect to https://konghq.com/install/ so maybe we can link directly here.

Reply:
👍